### PR TITLE
Readonly color input

### PR DESCRIPTION
**Pull request by kristofferj**
#882

### CODE REVIEW

File: packages/@sanity/color-input/src/ColorInput.js [hunk 1]
```diff
@@ -46,6 +46,7 @@ export default class ColorInput extends PureComponent {
 
   emitSetColor = nextColor => {
     const {onChange, type} = this.props
+    console.log('change')
```
Comment (bjoerge) [inline packages/@sanity/color-input/src/ColorInput.js]: This should be removed

File: packages/@sanity/color-input/src/ColorPicker.js [hunk 2]
```diff
@@ -4,41 +4,47 @@ import Button from 'part:@sanity/components/buttons/default'
 import {ColorWrap, Checkboard, Saturation, Hue, Alpha} from 'react-color/lib/components/common'
 import ColorPickerFields from './ColorPickerFields'
 import TrashIcon from 'part:@sanity/base/trash-icon'
+import invertColor from 'invert-color'
```
Comment (bjoerge) [inline packages/@sanity/color-input/src/ColorPicker.js]: This doesn't seem to be in use?

File: packages/@sanity/color-input/src/ColorPicker.js [hunk 3]
```diff
@@ -4,41 +4,47 @@ import Button from 'part:@sanity/components/buttons/default'
...
 import styles from './ColorPicker.css'
 
-const ColorPicker = ({width, rgb, hex, hsv, hsl, onChange, onUnset, disableAlpha, renderers}) => {
+const ColorPicker = ({width, rgb, hex, hsv, hsl, onChange, onUnset, disableAlpha, renderers, readOnly}) => {
+  const invertedHue = ((hsl.h + 180) % 360) < 0 ? 360 + hsl.h : hsl.h;
```
Comment (bjoerge) [inline packages/@sanity/color-input/src/ColorPicker.js]: `invertedHue` doesn't seem to be in use here?

File: packages/@sanity/color-input/src/ColorInput.js (hunk 1 above)
Comment (kristofferj) [reply packages/@sanity/color-input/src/ColorInput.js]: oupsie

File: packages/@sanity/color-input/src/ColorPicker.js (hunk 2 above)
Comment (kristofferj) [reply packages/@sanity/color-input/src/ColorPicker.js]: Yeah. Tried another approach first. Left something behind :)

**Pull request merged**